pkgs/by-name/sh/shh/package.nix

```
@@ -2,12 +2,21 @@
   lib,
   rustPlatform,
   fetchFromGitHub,
+  nix-update-script,
+  fetchpatch,
+  installShellFiles,
   python3,
   strace,
   systemd,
   iproute2,
+  stdenv,
+  enableDocumentationFeature ? true,
+  enableDocumentationGeneration ? true,
 }:
-
+let
+  isCrossCompile = !(stdenv.buildPlatform.canExecute stdenv.hostPlatform);
+  isNativeDocgen = !isCrossCompile && enableDocumentationFeature;
```

```suggestion
  isNativeDocgen = !(stdenv.buildPlatform.canExecute stdenv.hostPlatform) && enableDocumentationFeature;
```

`isCrossCompile` is misleading